src/Core/Entities/Organization.cs

```diff
@@ -70,6 +70,8 @@ public class Organization : ITableObject<Guid>, ISubscriber, IStorable, IStorabl
         public DateTime RevisionDate { get; set; } = DateTime.UtcNow;
         public int? MaxAutoscaleSeats { get; set; } = null;
         public DateTime? OwnersNotifiedOfAutoscaling { get; set; } = null;
+        [MaxLength(30)]
+        public string CloudBillingSyncKey { get; set; }
```

Comment:
So, just throwing this out into the ether. We are already adding a table for storing the billing sync key that is valid to be consumed. What if we also store the one they want consumed in that table?

Reply:
I thought about this, too, but I really dislike using things for multiple purposes.

Perhaps if we had a specific api key type...

Reply:
Yeah it's definitely a bit weird but I was thinking just that currently I have a `Default` and `BillingSync` type but we could have a third `CloudBillingSync`? Name could for sure be work shopped but it might be nicer to just have one database migration in all this.